README.md

```diff
@@ -302,111 +302,437 @@ document="content://..."
 ## API
 
 ### RNPdftron
+- [initialize](#initialize)
```

Comment:
hard to maintain, remove

Reply:
I think this comment is referring to the entire section, not line 305 specifically.

Reply:
yes, the entire link section